Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+        Returns
+        -------
+        None
+        """
+
+
+    class fileHandlingMTSET():
+        """
+        
+        """
+
+        def __init__(self, *args) -> None:
```

Ruff (0.8.2)

Missing type annotation for `*args`

(ANN002)